website/docs/docs/cloud/manage-access/set-up-sso-azure-active-directory.md

```diff
@@ -144,7 +144,7 @@ To complete setup, follow the steps below in the dbt Cloud application.
 | ----- | ----- |
 | **Log&nbsp;in&nbsp;with** | Azure AD Single Tenant |
 | **Client&nbspID** | Paste the **Application (client) ID** recorded in the steps above |
-| **Client&nbsp;Secret** | Paste the **Client Secret** (remember to use the Secret Value instead of the Secret ID) recorded in the steps above |
+| **Client&nbsp;Secret** | Paste the **Client Secret** (remember to use the Secret Value instead of the Secret ID) recorded in the steps above; **Note:** When the client secret expires, an Azure AD admin will have to generate a new one to be pasted into dbt Cloud for uninterrupted application access |
```

Comment:
Adding line break to increase visibility of the note
```suggestion
| **Client&nbsp;Secret** | Paste the **Client Secret** (remember to use the Secret Value instead of the Secret ID) recorded in the steps above; <br />**Note:** When the client secret expires, an Azure AD admin will have to generate a new one to be pasted into dbt Cloud for uninterrupted application access. |
```